### PR TITLE
update all examples as I could

### PR DESCRIPTION
**Pull request by ThierryBerger**
Related to https://github.com/bevyengine/bevy/pull/8745 : Isolated example changes to focus on API changes, and impact on examples.

 Closed due to https://github.com/Vrixyz/bevy/pull/2#issuecomment-1735081975

### CODE REVIEW

File: examples/3d/anti_aliasing.rs [hunk 1]
```diff
@@ -45,14 +45,14 @@ fn modify_aa(
     let mut camera = commands.entity(camera_entity);
 
     // No AA
-    if keys.just_pressed(KeyCode::Digit1) {
+    if keys.just_pressed(Key::Character("1".into())) {
```
Comment (ThierryBerger) [inline examples/3d/anti_aliasing.rs]: I'm being inconsistent by sometimes relying on Key and sometimes on KeyCode ; I'm not sure what we want:
- relying on Key is more correct because the user has to enter the correct character from his keyboard
- relying on KeyCode is maybe more practical (on AZERTY `Key::Character("1".into())` is <kbd>shift</kbd> + <kbd>1</kbd> where `KeyCode::Digit1` is just that key <kbd>1</kbd>)

File: examples/input/keyboard_input.rs [hunk 2]
```diff
@@ -22,4 +22,7 @@ fn keyboard_input_system(keyboard_input: Res<Input<Key>>) {
     if keyboard_input.just_released("a") {
         info!("'A' just released");
     }
+    if keyboard_input.pressed("A") {
+        info!("capital 'A' is being pressed");
+    }
```
Comment (ThierryBerger) [inline examples/input/keyboard_input.rs:27]: More info: https://gist.github.com/Vrixyz/16e7d7eb204f31f768c96f59f4bfa40d

TLDR: difficult to cache a Key::Character with modifiers.

I think we'll want our cache to rely on `key_without_modifiers` rather than `logical_key`, and aside from that our `ReceivedCharacter` event can take the `logical_key` because it doesn't rely on caching.

It needs an update from winit to expose the key without modifiers though.

File: examples/3d/anti_aliasing.rs (hunk 1 above)
Comment (ThierryBerger) [reply examples/3d/anti_aliasing.rs]: Probably won't be an issue after https://github.com/Vrixyz/bevy/pull/2#discussion_r1230686076

File: examples/input/keyboard_input.rs (hunk 2 above)
Comment (ThierryBerger) [reply examples/input/keyboard_input.rs:27]: https://github.com/Vrixyz/bevy/pull/3 should fix it.

**Comment by ThierryBerger** [comment]
I remove this PR in favor of https://github.com/Vrixyz/bevy/pull/3, because:

- ergonomics improvements had a lot of impact on examples.
- we decided on https://github.com/bevyengine/bevy/pull/8745#issuecomment-1718264284 to use KeyCode as much as possible in examples, so this PR is mostly incorrect.
